src/08-tasks/tasks.apib

```diff
@@ -48,6 +48,8 @@ Get a list of tasks.
                             + user
                             + team
                     + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
+                + milestone_id: `6fac0bf0-e803-424e-af67-76863a3d7d16` (string, nullable)
```

Comment:
B: Should be a `milestone` object instead of `milestone_id`. Same for `deal` and same for `.info`.